bin/ir_resolver.ml

```diff
@@ -76,10 +76,10 @@ let key k default =
   let doc = Irmin.Private.Conf.doc k in
   let docs = Irmin.Private.Conf.docs k in
   let docv = Irmin.Private.Conf.docv k in
-  let conv = pconv (Irmin.Private.Conf.conv k) in
+  let mk = pconv (Irmin.Private.Conf.conv k) in
```

Comment:
Well if you want to support both 0.9 and 1.0 yes. But then what's the point ? I seems that all incompatible packages are being quickly updated. I rather fear that you'll forget about this and that the package will break whenever I'll make the type abstract.